CHANGELOG-1.x.md

```diff
@@ -1,3 +1,5 @@
+# V1.4.4
+*Post release update to v1.4 branch (v1.4.3) ([#810](https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/810), [@Ashley-wenyizha](https://github.com/Ashley-wenyizha))
```

Comment:
Nit : trivial change and not worthy to mention in the release, can remove

Not Nit : Add efs-utils version change in this new release and brief mention why the release of this specific version